src/templates/app/user/security/account_closure.jsx

```diff
@@ -0,0 +1,150 @@
+import React             from 'react';
+import {
+    FormRow,
+    SubmitButton }       from '../../../_common/components/forms.jsx';
+import Loading           from '../../../_common/components/loading.jsx';
+import { SeparatorLine } from '../../../_common/components/separator_line.jsx';
+    
+const AccountClosure = () => (
+    <React.Fragment>
+        <div id='msg_main' className='center-text gr-gutter gr-padding-10 invisible'>
+            <h1>{it.L('Account closure confirmed')}</h1>
+            <p className='notice-msg'>
+                {it.L('Accounts closed successfully. A confirmation email will be sent to your email.')}
+                <br />
+                {it.L('This page will redirect to the [_1] homepage after 10 seconds.', it.website_name)}
+            </p>
+        </div>
+        <div id='closure_container'>
+            <div id='main_header' className='gr-padding-30'>
+                <h1 id='heading'>{it.L('Account Closure')}</h1>
+                <p>{it.L('Closing your [_1] accounts involves closing all open positions in your accounts, withdrawing your funds, and deactivating your accounts with [_1].', it.website_name)}</p>
+            </div>
+
+            <div className='gr-no-gutter' id='closure_description'>
+                <h2 className='primary-color'>{it.L('What would you like to do?')}</h2>
+                <fieldset>
+                    <div className='gr-padding-20 gr-gutter-left gr-gutter-right'>
+                        <ClosureDescription
+                            id='change-fiat'
+                            title={it.L('Change my account currency')}
+                            subtitle={it.L('[_1]Change your fiat currency[_2] to any of the following:', `<a href="${it.url_for('user/accounts')}">`, '</a>')}
```

Comment:
did we decide to leave it as is with no changes?

Reply:
Yeah, they were okay with leaving all the messages without any checks

Reply:
in addition, as per discussion with lucas and jeya. and we will redirect the link to manage account if they want to change currency. the message of they don't have or they have the account will be shown there already in the manage account page.